docs: fix typo (#8129)
nuxt/framework: docs/content/1.getting-started/2.installation.md

## New Project

> Open a terminal or from [Visual Studio Code](https://code.visualstudio.com/) open an [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal) and use the following command to create a new starter project:

Reviewer: ```suggestion
Open a terminal (if you're using [Visual Studio Code](https://code.visualstudio.com/), you can open an [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal)) and use the following command to create a new starter project:
```
